airflow/providers/amazon/aws/hooks/base_aws.py

```diff
@@ -62,16 +63,14 @@ class AwsBaseHook(BaseHook):
 
     def __init__(
             self,
-            aws_conn_id="aws_default",
+            aws_conn_id: str = "aws_default",
```

Comment:
Should not that be Optional[str] ?

Reply:
Optional is required only when default value is `None`